pkg/scalers/aws_sqs_queue_scaler_test.go

```diff
@@ -28,6 +28,12 @@ const (
 	testAWSSQSBadDataQueueURL = "https://sqs.eu-west-1.amazonaws.com/account_id/BadData"
 )
 
+var testSQSResolvedEnv = map[string]string{
+	"awsAccessKeyId":     testAWSSQSAccessKeyID,
+	"awsSecretAccessKey": testAWSSQSSecretAccessKey,
+	"QUEUE_URL":          testAWSSQSProperQueueURL,
+}
+
 var testAWSSQSAuthentication = map[string]string{
```

Comment:
If I remove the keys, then the test fails with `aws_sqs_queue_scaler_test.go:279: Expected success because properly formed queue loaded from env got error, awsAccessKeyID not found`. Perhaps it might be better if I rename `testAWSSQSAuthentication` to `testAWSSQSResolvedEnv`?

Reply:
No, keep them both. Refer to the other comment I made below.

Reply:
No, keep them both. Refer to the other comment I made below.